lib/shared.d.ts

```diff
@@ -1,4 +1,3 @@
-import type { SemVer } from "semver";
 export interface parsed {
```

Comment:
Can we rename this to `Pep440Version` in next PR?

Reply:
Took the name after the constant in parse function.
https://github.com/renovatebot/pep440/blob/eefd68be4df448ac6ce3ab370317e6e94df95ee9/lib/version.js#L54-L75
So we can change the name for sure.